src/commands/protocols/common/epoch-check-command.js

```diff
@@ -124,12 +126,39 @@ class EpochCheckCommand extends Command {
         for (const serviceAgreement of eligibleAgreementForSubmitCommit) {
             if (scheduleSubmitCommitCommands.length >= maxTransactions) break;
 
+            const neighbourhood = await this.shardingTableService.findNeighbourhood(
+                blockchain,
+                serviceAgreement.keyword,
+                r2,
+                serviceAgreement.hashFunctionId,
+                serviceAgreement.scoreFunctionId,
+                true,
+            );
+
+            let neighbourhoodEdges = null;
+            if (serviceAgreement.scoreFunctionId === 2) {
+                neighbourhoodEdges = await this.shardingTableService.getNeighboorhoodEdgeNodes(
```

Comment:
Let's keep American version `neighborhood` consistently everywhere
```suggestion
                neighborhoodEdges = await this.shardingTableService.getNeighborhoodEdgeNodes(
```